src/beautifiers/prettydiff.coffee

```diff
@@ -109,6 +110,8 @@ module.exports = class PrettyDiff extends Beautifier
           lang = "scss"
         when "TSS"
           lang = "tss"
+        when "Golang Template"
+          lang = "gohtml"
```

Comment:
@prettydiff i think need change            ` go        : "html",` =>             `gohtml       : "html",`

Because .go for golang and .gohtml for golang template file.